Session/Conversations/ConversationVC.swift

```diff
@@ -1647,6 +1657,22 @@ final class ConversationVC: BaseVC, LibSessionRespondingViewController, Conversa
             completion?()
         }
     }
+    
+    private func makeInputViewFirstResponder() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
```

Comment:
Might be good to add a comment indicating that the delay is needed because without it there is a glitch in the keyboard appearance animation